lib/deserializers.php

```diff
@@ -161,3 +161,31 @@ function elementList(Reader $reader, $namespace = null) {
     return $values;
 
 }
+
+/**
+ * @param Reader $reader
+ * @param $popo
+ * @param $namespace
+ * @return null|$popo
+ */
+function popoObject(Reader $reader, $popo, $namespace) {
+    if ($reader->isEmptyElement) {
+        $reader->next();
+        return null;
+    }
+
+    $reader->read();
+    do {
+
+        if ($reader->nodeType === Reader::ELEMENT && $reader->namespaceURI == $namespace) {
```

Comment:
Could this function use use `keyValue` instead? It's one more layer of indirection, but it makes the implementation of popoObject a lot simpler (which can benefit people reading the source).